examples/player/simid_player.js

```diff
@@ -55,6 +56,13 @@ class SimidPlayer {
      */
     this.adComplete_ = adComplete;
```

Comment:
Does this need a JSDoc?

Reply:
i think it can be confusing what adComplete_ is without the jsdoc. I could be wrong